app/src/main/java/io/apicurio/registry/storage/impl/sql/AbstractSqlRegistryStorage.java

```diff
@@ -301,7 +300,7 @@ private void initializeDatabase(Handle handle) {
         log.debug("---");
 
         statements.forEach(statement -> {
-            log.debug(statement);
+            log.info(statement);
```

Comment:
Is this necessary?

Reply:
No this was a mistake to include, thanks for catching that.